### ISSUE TITLE
Assets.xcassets -> App Icon Warnings

### ISSUE THREAD

**Issue by masarapmabuhay** [chrisamanse/LIKHA-Guide#8]
Good news! I've gotten the LIKHA to build successfully. It ran on the simulator as well. However, I noticed that there are still Warnings related to the App Icon.

<img width="255" alt="screen shot 2016-10-01 at 6 49 21 pm" src="https://cloud.githubusercontent.com/assets/344066/19013596/211d9644-8808-11e6-8e61-be4b17573c34.png">

If we look at Assets.xcassets -> App Icon, things seem to be OK.
<img width="794" alt="screen shot 2016-10-01 at 6 48 53 pm" src="https://cloud.githubusercontent.com/assets/344066/19013603/4e086580-8808-11e6-80bc-8de0ee7d541b.png">

Am I missing a step here?

**Comment by chrisamanse** [comment]
Can you try regenerating your AppIcon images using Iconizer? The images looks like there in the wrong sizes and some images are not generated. Also, make sure to tick only iPhone, iPad, and Export as Combined Asset checkboxes in Iconizer.

**Comment by masarapmabuhay** [comment]
Oui, I have regenerated the AppIcon images using Iconizer as per your instruction multiple times. Should I also have copy-pasted Contents.json? Merci bien!

**Comment by chrisamanse** [comment]
The steps are mentioned in this [comment](https://github.com/usbong/LIKHA-Guide/issues/2#issuecomment-250387091). There's no need to copy-paste the JSON file. All you need to do is to replace `AppIcon.appiconset` with the one generated by `Iconizer`.

**Comment by masarapmabuhay** [comment]
Salut! It seems that the Warnings still persist even after I went through the steps you laid out in your comment, and regenerated the AppIcon images again. The dimensions also seem double the correct size, e.g. 200x200, instead of 100x100. C'est étrange!

**Comment by chrisamanse** [comment]
Can you send me a `.zip` file of your `AppIcon.appiconset` folder, and the 1024x1024 icon used in Iconizer? Also can you send me the `LIKHA.xcodeproj/project.pbxproj`. To go inside `.xcodeproj` files, right-click on it, then choose _Show Package Contents_. This will help me inspect your current project configuration.

**Comment by masarapmabuhay** [comment]
Salut! Here are the 3 files that you requested:
1) [AppIcon.appiconset.zip](https://github.com/usbong/LIKHA-Guide/files/504935/AppIcon.appiconset.zip)
2) ![none_d1s1-1024x1024](https://cloud.githubusercontent.com/assets/344066/19020695/bbdbd2ce-88e2-11e6-82d0-3aeae424d893.png)
3) [LIKHA.xcodeproj.zip](https://github.com/usbong/LIKHA-Guide/files/504937/LIKHA.xcodeproj.zip)

Merci beaucoup!

**Comment by chrisamanse** [comment]
1) The app icons really are twice the size expected, and some images are duplicates (the `*-1.png` files).

2) I tried generating the images using Iconizer versions 2.2 and 2.2.1, and both of them are generating  images without any issues. Here's my 
[AppIcon.appiconset.zip](https://github.com/usbong/LIKHA-Guide/files/504985/AppIcon.appiconset.zip).

3) I found no issues in the project configuration.

I think the problem is with your Iconizer app. This may be an issue with the Iconizer app. Try deleting first the old `AppIcon.appiconset` generated by Iconizer, then generate the app icons again. There may be a problem with the overwriting process in Iconizer.

If the issue still persists, feel free to use the `AppIcon.appiconset` I attached. And you may also report the issue in the GitHub repo of [Iconizer](https://github.com/raphaelhanneken/Iconizer).

**Comment by masarapmabuhay** [comment]
Salut! I downloaded Iconizer version 2.2.1, and found that the images are still double in size. I've deleted the previous `AppIcon.appiconset` generated by Iconizer, and generated the app icons again, as you recommended. Unfortunately, the issue still persists.

I may have to report the issue in the GitHub repo of Iconizer then.

Toutefois, merci beaucoup!

**Comment by chrisamanse** [comment]
I've created pull request raphaelhanneken/iconizer#7 in Iconizer that should fix the resizing of images.

Although, I haven't tested it in a Mac with a Retina Screen yet. You may test it using this app I built with the fix:

[Iconizer.zip](https://github.com/usbong/LIKHA-Guide/files/505374/Iconizer.zip)

**Comment by masarapmabuhay** [comment]
Salut! I tested your Iconizer.zip. Unfortunately, the problem still persists. The dimensions of the images are still double in size. 

<img width="253" alt="screen shot 2016-10-04 at 9 34 05 am" src="https://cloud.githubusercontent.com/assets/344066/19059884/e82a3f76-8a15-11e6-9347-0b756ecda8a6.png">

I also noticed that it has other Warnings, e.g. "A 60x60@2x app icon is required for iPhone apps targeting iOS 7.0 and later".

Also, your updated Iconizer can already generate "ipad-pro-appicon@1x.png", but does not generate "ipadpro-appicon@2x.png".

Merci beaucoup comme toujours!

**Comment by masarapmabuhay** [comment]
Salut! I'm now moving forward with the guide using the `AppIcon.appiconset.zip` that you gave. I noticed though that `ipadpro-appicon@2x.png` is missing. Perhaps this particular image is really not generated? Merci beaucoup!

**Comment by chrisamanse** [comment]
Technically, the `ipadpro-appicon@1x.png` should be `ipadpro-appicon@2x.png`. But if you also copy the generated `Contents.json` inside the `AppIcon.appiconset` folder (recommended), the iPad Pro App Icon for @2x should map to the `*@1x.png` file:

``` json
{
      "size" : "83.5x83.5",
      "filename" : "ipad-pro-appicon@1x.png",
      "expected-size" : "167",
      "idiom" : "ipad",
      "scale" : "2x"
},
```

This is a file naming bug of Iconizer, but there should be no issues if you copy the entire `.appiconset` folder.

**Comment by masarapmabuhay** [comment]
D'accord! So that means I'll also have to copy `Contents.json` then. I was under the impression that I didn't have to copy it. Merci bien!

p.s. the updated Iconizer you sent still seems to produce the same Warnings as the earlier version.
